README.md

```diff
@@ -68,6 +68,8 @@ We have tried this in the following configurations:
 | Monitor         | Laptop                                    | OS                 | Cable   | Result                             |
 | --------------- | ----------------------------------------- | ------------------ | ------- | ---------------------------------- |
 | Dell P2720DC    | MacBook Pro (Retina, 13-inch, Early 2015) | Catalina (10.15.7) | HDMI    | :heavy_check_mark: Switched to RGB |
+| --------------- | ----------------------------------------- | ------------------ | ------- | ---------------------------------- |
```

Comment:
```suggestion
```
I think in the Github markdown flavor only the first line gets separated by the `---`, see https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown

Reply:
Indeed, thank you